compiler/rustc_lint/src/lints.rs

```diff
@@ -2066,6 +2066,7 @@ pub enum InvalidAsmLabel {
         missing_precise_span: bool,
     },
     #[diag(lint_invalid_asm_label_binary)]
+    #[help]
```

Comment:
What do I need to do to make this propagate to https://github.com/rust-lang/rust/blob/5753b3067662e17a69b54b9418dbc37b73769a84/tests/ui/asm/binary_asm_labels.stderr#L1-L8?

Reply:
It should work. The test is gated on `only-x86_64`, so if you weren't on x86_64 it wouldn't be blessed.

Reply:
Ah, maybe I should try running it on an actual x86 machine